test/test_jit.py

```diff
@@ -7732,12 +7745,24 @@ def new_fn(*tensors_):
     return new_fn, [arg for arg in args if isinstance(arg, torch.Tensor)]
 
 
+def assert_fully_differentiable_graph(graph):
+    nodes = list(graph.nodes())
+    if not (len(nodes) == 1 and 'prim::DifferentiableGraph' in str(nodes[0])):
```

Comment:
can we check `nodes[0].kind() == 'prim::DifferentiableGraph` instead? Feels less hacky

Reply:
Oh I don't think I'm actually using this function. I'll delete if if I'm not (otherwise, your suggestion sounds good)